analyzer/java/com/engflow/bazel/invocation/analyzer/dataproviders/FlagValueExperimentalProfileIncludeTargetLabel.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 EngFlow Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.engflow.bazel.invocation.analyzer.dataproviders;
+
+import com.engflow.bazel.invocation.analyzer.core.Datum;
+import java.util.Objects;
+
+/**
+ * Whether the Bazel flag `--experimental_profile_include_target_label` was enabled when the Bazel
+ * profile was generated.
+ *
+ * @see <a
+ *     href="https://bazel.build/reference/command-line-reference#flag--experimental_profile_include_target_label">Bazel
+ *     Command-Line Reference</a>
+ */
+public class FlagValueExperimentalProfileIncludeTargetLabel implements Datum {
```

Comment:
```suggestion
public final class FlagValueExperimentalProfileIncludeTargetLabel implements Datum {
```

Reply:
None of our `Datum`s so far have been final. What's the reason you think it should be `final`?

Reply:
I'd prefer to keep these non-final, as I do see a use-case for extending `Datum`s.